workflows/repeatmasking/RepeatMasking-Workflow.ga

```diff
@@ -0,0 +1,168 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "format-version": "0.1",
+    "license": "MIT",
+    "release": "0.1",
+    "name": "Workflow constructed from history 'RepeatMasking'",
```

Comment:
Could you give this workflow a nice name, like "Repeat masking with RepeatModeler and RepeatMasker"